vectordatasource/transform.py

```diff
@@ -2952,20 +2895,13 @@ def _merge_features_by_property(features, drop_props_fn=None,
             new_features.append((shapes[0], orig_props, fid))
```

Comment:
I expect that should be the behavior, because the `orig_props` are used here, ie a copy before the `props` are modified. I should probably add a test to verify.